koocook_core/models/review.py

```diff
@@ -47,6 +60,10 @@ class Comment(SerialisableModel, models.Model):
         blank=True,
     )
 
+    def create_empty_aggregate_rating(**kwargs) -> 'AggregateRating':
+        """Creates an empty aggregate rating"""
+        return AggregateRating.objects.create(rating_value=0, rating_count=0, **kwargs)
+
```

Comment:
I don't think this is needed here?

Reply:
Totally

Reply:
Is using `AggregateRating.create_empty()` and still having this function the result of using changes' in `test/models`?

Reply:
I guess so.

Reply:
Just remove it then. It's going into dev so ut should be right. I don't care it's supposed to be changed anymore.